spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -1,5 +1,32 @@
 describe 'orifice-calculator-viewmodel-spec', ->
-  viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+  viewModel = null
+  field = ko.observable()
+
+  beforeEach ->
+    viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+
+  itBehavesLikeMandatoryField = () ->
+    it 'has mandatory field validation', ->
+      field('')
+      expect(field.isValid()).toEqual false
+
+  itBehavesLikeIntegerField = () ->
+    it 'should only accept integer', ->
+      field(123)
+      expect(field.isValid()).toEqual true
```

Comment:
You can/should check that the viewModel is not valid and the errors include the correct message